docs/reference/lambda-support.md

```diff
@@ -0,0 +1,435 @@
+---
+mapped_pages:
+  - https://www.elastic.co/guide/en/apm/agent/python/current/lambda-support.html
+---
+
+# Monitoring AWS Lambda Python Functions [lambda-support]
+
+The Python APM Agent can be used with AWS Lambda to monitor the execution of your AWS Lambda functions.
+
+```
+Note: The Centralized Agent Configuration on the Elasticsearch APM currently does NOT support AWS Lambda.
+```
+
+
+## Prerequisites [_prerequisites]
+
+You need an APM Server to send APM data to. Follow the [APM Quick start](docs-content://solutions/observability/apps/get-started-with-apm.md) if you have not set one up yet. For the best-possible performance, we recommend setting up APM on {{ecloud}} in the same AWS region as your AWS Lambda functions.
```

Comment:
This is pointing to the apm server doc while we currently link to the observability apm page
https://www.elastic.co/guide/en/observability/8.17/apm.html

Reply:
```suggestion
You need an APM Server to send APM data to. Follow the [APM Quick start](docs-content://solutions/observability/apps/application-performance-monitoring-apm.md) if you have not set one up yet. For the best-possible performance, we recommend setting up APM on {{ecloud}} in the same AWS region as your AWS Lambda functions.
```